app/helpers/employees_helper.rb

```diff
@@ -1,4 +1,26 @@
-require 'attendance'
-
 module EmployeesHelper
+  def not_team_task?(task)
+    !task.assignable_type.eql? 'Team'
+  end
+
+  def team_task?(task)
+    return false if task.eql? -1
```

Comment:
what's '-1', should be a constant

Reply:
was using it before with previous logic, now its not used, i'll remove them